presto-main/src/main/java/com/facebook/presto/sql/planner/iterative/rule/ImplementFilteredAggregations.java

```diff
@@ -20,6 +20,8 @@
 import com.facebook.presto.sql.planner.plan.AggregationNode;
```

Comment:
Please add to the commit message information about rule application space, that this only work only for default aggregation.